mono/metadata/cominterop.c

```diff
@@ -2359,12 +2359,24 @@ cominterop_get_managed_wrapper_adjusted (MonoMethod *method)
 
 	if (!MONO_TYPE_IS_VOID (sig->ret)) {
 		if (!preserve_sig) {
-			MonoClass *rclass = mono_class_from_mono_type_internal (sig->ret);
+			int pos_null;
+			MonoClass *rclass;
+
+			mono_mb_emit_stloc (mb, retval);
+			mono_mb_emit_ldarg (mb, sig_native->param_count-1);
+			pos_null = mono_mb_emit_branch (mb, CEE_BRFALSE);
+
+			mono_mb_emit_ldarg (mb, sig_native->param_count-1);
```

Comment:
Spaces around binary operators please -- `param_count - 1` and above.

Reply:
Yep, that was copied from the previous code. Fixed now.